src/App.tsx

```diff
@@ -1,18 +1,25 @@
-import React from 'react';
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
 
 type Move = "rock" | "paper" | "scissors";
 const MOVES: readonly Move[] = ["rock", "paper", "scissors"];
 
 function App() {
-  const [computerMove, setComputerMove] = React.useState<Move>("rock");
+  const [computerMove, setComputerMove] = useState<Move>(() => 
+    MOVES[Math.floor(Math.random() * MOVES.length)]
+  );
+  const [playerMove, setPlayerMove] = useState<Move | null>(null);
+
   return (
     <div className="App">
-      Computer Move: {computerMove}
+      <p>Computer Move: {computerMove}</p>
+      <p>Player Move: {playerMove ? playerMove : ""}</p>
```

Comment:
React allows rendering nulls, but you if didn't want to do that you can still improve on this, by using the nullish coalescing operator:

```suggestion
      <p>Player Move: {playerMove ?? ""}</p>
```